plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -371,30 +353,6 @@ void startDevServer(String mode, DefaultRunConfiguration devServerRunConfigurati
     }
   }
 
-  @VisibleForTesting
-  static void putAppEngineApiSdkJarIntoApps(List<File> appDirectories,
-      ILibraryRepositoryService repositoryService) throws CoreException {
-    try {
-      Library library = CloudLibraries.getLibrary("appengine-api");
-      for (LibraryFile libraryFile : library.getAllDependencies()) {
-        Artifact artifact =
-            repositoryService.resolveArtifact(libraryFile, new NullProgressMonitor());
-        Path artifactPath = artifact.getFile().toPath();
-
-        for (File appDirectory : appDirectories) {
-          Path libDirectory = appDirectory.toPath().resolve("WEB-INF/lib");
-          libDirectory.toFile().mkdirs();
-          if (!appEngineApiSdkJarExists(libDirectory)) {
-            Files.copy(artifactPath, libDirectory.resolve(artifactPath.getFileName()));
-          }
-        }
-      }
-    } catch (IOException e) {
-      String message = Messages.getString("cannot.copy.appengine.api.sdk.jar");
-      throw new CoreException(StatusUtil.error(LocalAppEngineServerBehaviour.class, message, e));
-    }
-  }
-
   @VisibleForTesting
   static boolean appEngineApiSdkJarExists(Path directory) {
```

Comment:
Most likely this one can be removed too.

Reply:
Done